README.md

```diff
@@ -110,15 +110,16 @@ docker run \
   --network=host \
   --privileged \
   --workdir=/home/admin \
-  --env=SETUP_MODE=combined \
+  --env=SETUP_MODE=COMBINED \
   --env=MYSQL_HOST=<mysql_host> \
   --env=MYSQL_PORT=<mysql_port> \
   --env=MYSQL_USER=<mysql_user> \
   --env=MYSQL_PASSWORD=<mysql_password> \
   --detach=true \
   apecloud/myduckserver:latest
 ```
-
+## Try on Cloud Vendors
```

Comment:
"Try on Cloud Vendors" -> "Connect to Cloud MySQL"

Reply:
fixed